experimental/auth-logic/endorsement-release/input_policy_examples/oak_endorsement_policy.auth_logic

```diff
@@ -0,0 +1,40 @@
+"OakFunctionsLoaderTeam" says {
+    // The product team can change this line to pick how to accept outputs from 
+    // trusted builders. For example:
+    // - one team might only trust github actions, but not any other builder
+    // - another team might be OK with getting output from either builder
+    // - another team might insist on getting the same output from both builders
+    // - another team might want the same output from k out of n trusted builders
+    // Product teams will usually not want to write much authorization logic. 
+    // So we offer policy principals that encode these common options. The 
+    // product team can pick one of these options by _delegating_ to principals
+    // that encodes one of these policy choices. Product teams with different needs
+    // can also choose not to delegate to one of these policies and write a complete
+    // policy on their own
+
+    "GithubActionsPolicy" canSay BuildPolicyAllowRelease("oak_functions_loader:0f2189703c57845e09d8ab89164a4041c0af0a62::Binary", any_hash).
```

Comment:
Eventually I don't even think we need the entire `oak_functions_loader:0f2189703c57845e09d8ab89164a4041c0af0a62::Binary` thing, in principle just `sha256:0f2189703c57845e09d8ab89164a4041c0af0a62` should be sufficient, right?